### PR TITLE
Drukhari faction

### PR DESCRIPTION
**Pull request by Orta15a**
Added Drukhari faction secondaries

### CODE REVIEW

File: 9e/factions.json [hunk 1]
```diff
@@ -307,6 +307,52 @@
...
+		{
+			"faction": "DRUKHARI",
+			"name": "Drukhari",
+			"categories": [
+				{
+					"name": "Purge the Enemy",
+					"objectives": [
+						{
+							"id": "EW.09.01.01",
+							"name": "Take them alive !",
```
Comment (Zyllos) [inline 9e/factions.json]: Capitalize Them and Alive

File: 9e/factions.json [hunk 2]
```diff
@@ -307,6 +307,52 @@
...
+							"description": "[i]Progressive Objective[/i]\nIf you select this objective, you score victory points at the end of the battle for each of the following that apply: \n- Score 3 victory points if any CHARACTER or MONSTER units were destroyed by a melee attack made by a DRUKHARI unit from your army during that battle round.\n- Score 1 victory point if any unit (excluding CHARACTER, MONSTER or VEHICLE units) were destroyed by a melee attack made by a DRUKHARI unit from your army during that battle round."
+						}
+					]
+				},
+				{
+					"name": "Purge the Enemy",
+					"objectives": [
+						{
+							"id": "EW.09.02.01",
+							"name": "Beasts for the arenas",
```
Comment (Zyllos) [inline 9e/factions.json]: Capitalize Arenas

File: 9e/factions.json [hunk 3]
```diff
@@ -307,6 +307,52 @@
...
+							"description": "[i]End Game Objective[/i]\nScore 3 victory points at the end of the battle for each enemy MONSTER, CAVALRY or BEAST unit that was destroyed by a melee attack made by a <WYCH CULT> unit from your army. If that enemy unit was TITANIC, score 5 victory points for it instead."
+						}
+					]
+				},
+				{
+					"name": "No Mercy, No Respite",
+					"objectives": [
+						{
+							"id": "EW.09.03.01",
+							"name": "Fear and terror",
```
Comment (Zyllos) [inline 9e/factions.json]: Capitalize Terror

File: 9e/factions.json [hunk 4]
```diff
@@ -307,6 +307,52 @@
...
+							"description": "[i]Progressive Objective[/i]\nScore 1 victory points at the end of the battle round for each model that fled from any enemy unit during that battle round."
+						}
+					]
+				},
+				{
+					"name": "Battlefield Supremacy",
+					"objectives": [
+						{
+							"id": "EW.09.04.01",
+							"name": "Herd the prey",
```
Comment (Zyllos) [inline 9e/factions.json]: Capitalize Prey

**Review by Zyllos — COMMENTED** [review]
I would update the capitalization of these objective names.

**Comment by Orta15a** [comment]
Added FAQ update on latest commit.

File: 9e/factions.json (hunk 4 above)
Comment (Orta15a) [reply 9e/factions.json]: Done on latest commit

File: 9e/factions.json (hunk 3 above)
Comment (Orta15a) [reply 9e/factions.json]: Done on latest commit

File: 9e/factions.json (hunk 2 above)
Comment (Orta15a) [reply 9e/factions.json]: Done on latest commit

File: 9e/factions.json (hunk 1 above)
Comment (Orta15a) [reply 9e/factions.json]: Done on latest commit

**Comment by Orta15a** [comment]
Added Admech faction objectives

**Pull request merged**